src/test/java/net/snowflake/client/jdbc/telemetryOOB/TelemetryServiceIT.java

```diff
@@ -258,4 +294,31 @@ public void testSnowflakeSQLLoggedExceptionIBTelemetry() throws SQLException {
       assertThat("Communication error", e.getErrorCode(), equalTo(fakeErrorCode));
     }
   }
+
+  /**
+   * Test case for checking telemetry message for SnowflakeFeatureNotSupporteExceptions. In-band
+   * telemetry should be used. After running test, check for telemetry message in client_telemetry_v
+   * table.
+   *
+   * <p>Test is ignored because since responses are immediately sent asynchronously (before
+   * execution stops due to sqlexception being thrown) instead of being stored for awhile, it's
+   * difficult to pull out the telemetry logs and look at them before they're sent.
+   *
+   * @throws SQLException
+   */
+  @Ignore
+  @Test
+  public void testSqlFeatureNotSupportedExceptionIBTelemetry() throws SQLException {
+    // make a connection to initialize telemetry instance
+    Connection con = getConnection();
```

Comment:
Change this to `try(Connection con = getConnection()) { ... }` to ensure the connection is always closed.